test/util/fake-xml-http-request-test.js

```diff
@@ -1086,14 +1107,15 @@ if (typeof window !== "undefined") {
                 assert.isFalse(this.xhr.sendFlag);
             });
 
-            it("dispatches readystatechange event if sent before", function () {
+            it("does not dispatch readystatechange event if sent before", function () {
                 this.xhr.open("GET", "/");
```

Comment:
OK, that's fine, but could you please change the name to whatever makes sense to you and/or the casual reader? Just to avoid dummies like myself wondering what the test is supposed to mean in the future :-)